build/patches/Enable-incognito-custom-tab.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/CustomTabIncognitoManager.java b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/CustomTabIncognitoManager.java
+--- a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/CustomTabIncognitoManager.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/CustomTabIncognitoManager.java
+@@ -50,9 +50,10 @@ public class CustomTabIncognitoManager implements NativeInitObserver, Destroyabl
+
+     @Override
+     public void onFinishNativeInitialization() {
+-        assert ChromeFeatureList.isInitialized();
++        //assert ChromeFeatureList.isInitialized();
+         if (mIntentDataProvider.isIncognito()
+-                && ChromeFeatureList.isEnabled(ChromeFeatureList.CCT_INCOGNITO)) {
++                //&& ChromeFeatureList.isEnabled(ChromeFeatureList.CCT_INCOGNITO)) {
```

Comment:
This is incorrect, the feature can be enabled by default in `chrome/browser/flags/android/chrome_feature_list.cc` and leave this untouched.

Reply:
ok, I understand